include/aikido/perception/ObjectDatabase.hpp

```diff
@@ -10,8 +10,8 @@
 namespace aikido {
 namespace perception {
 
-/// Instantiation of ObjectDatabase that reads of JSON file containing the
-/// information that maps object keys to the object names and resources.
+/// Instantiation of ObjectDatabase (a.k.a. Asset Database) that reads of JSON file containing the
```

Comment:
Does it make sense to just call this class `AssetDatabase` rather than `ObjectDatabase`?